salt/modules/kmod.py

```diff
@@ -123,7 +123,16 @@ def available():
         salt '*' kmod.available
     '''
     ret = []
+
     mod_dir = os.path.join('/lib/modules/', os.uname()[2])
+
+    built_in_file = os.path.join(mod_dir, 'modules.builtin')
+    if os.path.exists(built_in_file):
+        with open(built_in_file, 'r') as f:
+            for line in f:
+                # Strip .ko from the basename
+                ret.append(os.path.basename(line)[:-4])
```

Comment:
is `-4` correct here? I assume you just want to strip off the `.ko` on the end, yes?

Reply:
-4 is correct: `.ko\n`. I can run `.rstrip()` on basename if that makes it easier to understand.

Reply:
Missed the newline! My fault. That makes sense.